test/last.js

```diff
@@ -1,19 +1,26 @@
 'use strict';
 
-const S = require ('..');
+const S = require ('./internal/sanctuary');
 
 const eq = require ('./internal/eq');
-
+const {Nil, Cons} = require ('./internal/List');
 
 test ('last', () => {
 
   eq (typeof S.last) ('function');
   eq (S.last.length) (1);
-  eq (S.show (S.last)) ('last :: Array a -> Maybe a');
+  eq (S.show (S.last)) ('last :: Foldable f => f a -> Maybe a');
 
   eq (S.last ([])) (S.Nothing);
   eq (S.last (['foo'])) (S.Just ('foo'));
   eq (S.last (['foo', 'bar'])) (S.Just ('bar'));
   eq (S.last (['foo', 'bar', 'baz'])) (S.Just ('baz'));
 
+  eq (S.last (Nil)) (S.Nothing);
+  eq (S.last (Cons ('foo') (Nil))) (S.Just ('foo'))
+  eq (S.last (Cons ('foo') (Cons ('bar') (Nil))))
+  (S.Just ('bar'));
+  eq (S.last (Cons ('foo') (Cons ('bar') (Cons ('baz') (Nil)))))
+     (S.Just ('baz'));
```

Comment:
Don't worry about line length in the test files. These assertions would read better as one-liners.

Reply:
🆗

Reply:
It looks as though you deleted some of the assertions. I was suggesting that you remove the line breaks rather than the whole lines. ;)